x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/hooks/setup_technology.ts

```diff
@@ -62,7 +62,6 @@ export const useAgentless = () => {
 export function useSetupTechnology({
   setNewAgentPolicy,
   newAgentPolicy,
-  updateAgentPolicies,
```

Comment:
@elastic/cloud-security-posture probably this file and the related test file is what triggered codeowners review. I removed the triggering of `updateAgentPolicies`, triggering `setNewAgentPolicy` suffices here from my testing. it was causing issues when switching between agent-based/agentless while also switching between new hosts/existing hosts

Reply:
Thanks for the additional context!